test_team1.py

```diff
@@ -27,6 +27,43 @@ def test_us_street_address(self):
         streetAddress = 'My street address is 1234 Invalid Input Drive, Los Angeles, CA, 271'
         self.assertFalse(find_us_street_address(streetAddress))
 
+    def test_credit_card_number(self):
+        # Test Valid
+        cardNum = '1234 5678 9012 3456'
+        self.assertFalse(find_credit_card_number(cardNum))
```

Comment:
Since this is a valid card number you should use self.assertTrue() because you would expect your function to return True.  The fact that it is returning False means that your regex is not working.  I would start with something much simpler like this
```
\d{4}[- ]\d{4}[- ]\d{4}[- ]\d{4}
```
and then, perhaps, make it more complicated to look for other types of credit cards like amex, but that's not a requirement.